include/bluetooth/services/fast_pair.h

```diff
@@ -36,15 +36,20 @@ size_t bt_fast_pair_adv_data_size(bool fp_discoverable);
  *
  * The buffer size must be at least @ref bt_fast_pair_adv_data_size.
  *
- * @param[out] adv_data		Pointer to the Bluetooth advertising data structure to be filled.
- * @param[out] buf		Pointer to the buffer used to store Fast Pair advertising data.
- * @param[in]  buf_size		Size of the buffer used to store Fast Pair advertising data.
- * @param[in]  fp_discoverable	Boolean indicating if device is Fast Pair discoverable.
+ * @param[out] adv_data		     Pointer to the Bluetooth advertising data structure to be
+ *                                   filled.
+ * @param[out] buf		     Pointer to the buffer used to store Fast Pair advertising data.
+ * @param[in]  buf_size		     Size of the buffer used to store Fast Pair advertising data.
+ * @param[in]  fp_discoverable	     Boolean indicating if device is Fast Pair discoverable.
+ * @param[in]  fp_show_ui_indication Boolean indicating if UI indication are to be shown on Fast
+ *                                   Pair Seeker device. This parameter is used only, if device is
+ *                                   not Fast Pair discoverable and the Account Key List is not
```

Comment:
`Fast Pair not discoverable` (I would treat it more as name of Fast Pair advertising type)